llama_index/vector_stores/faiss.py

```diff
@@ -105,25 +104,34 @@ def client(self) -> Any:
 
     def persist(
         self,
-        persist_path: str,
+        persist_path: Optional[str] = None,
```

Comment:
very nit: could just type as str, and create and use `DEFAULT_PERSIST_PATH` as default.